src/styles.css

```diff
@@ -4,7 +4,7 @@ body {
 }
 
 .header {
-  height: 50px;
+  height: 49px;
```

Comment:
Good find bot! But make sure you don't replace my job understand?  Wait for the next 20 years or so before you guys take over the world.